api/src/main/java/org/pmiops/workbench/api/CohortsController.java

```diff
@@ -239,6 +246,8 @@ public Cohort apply(org.pmiops.workbench.db.model.Cohort cohort) {
             String.format("Couldn't find cohort with name %s in workspace %s/%s",
                 request.getCohortName(), workspaceNamespace, workspaceId));
       }
+      cohortReview = cohortReviewDao.findCohortReviewByCohortIdAndCdrVersionId(cohort.getCohortId(),
```

Comment:
This call exists in CohortReviewServiceImple: https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java#L98 Does it make sense to you this service if some of the dao calls exist? Maybe.. maybe not?

Reply:
I didn't want the behavior that throws NotFoundException in the service when the review isn't found... it's fine if cohortReview is set to null here.

I guess I could catch NotFoundException, but that seems kludgier.

If we want all access to cohort reviews to go through the service, I could add another method that delegates to the DAO and doesn't throw if it's not found... let me know if you'd prefer that.

Reply:
So far the only places calling this method are the service and this, and they don't share any other logic; so I think we're OK for now.